packages/dm-core/src/components/ViewCreator/utils.ts

```diff
@@ -26,5 +26,7 @@ export const getType = (document: TGenericObject, viewConfig: TViewConfig) => {
     const target: TGenericObject = resolvePath(document, viewConfig.scope)
     return target.type
   }
+  if (Array.isArray(document)) return document[0].type
```

Comment:
The array could still be empty, though?

Reply:
Good catch, added missing `?` operator. Still, this is not a permanent solution